channels/printer/client/cups/printer_cups.c

```diff
@@ -405,6 +405,7 @@ FREERDP_API rdpPrinterDriver* freerdp_printer_client_subsystem_entry(void)
 		uniq_cups_driver->id_sequence = 1;
 		uniq_cups_driver->driver.AddRef(&uniq_cups_driver->driver);
```

Comment:
You should probably remove this in order not to double-count

Reply:
thx,